packages/core/src/scss/components/checkout/checkoutSteps/_checkoutSteps.scss

```diff
@@ -138,6 +138,21 @@
     min-width: 9rem;
 }
 
+.remove-checkout-step-numbers {
+    li.checkout-step {
+        div.stepHeader-counter.icon {
+            display: none;
+        }
+
+        .checkout-form,
+        div.stepHeader-body,
+        div.checklist-skeleton,
+        h2.stepHeader-title {
+            margin-left: 0;
+        }
```

Comment:
Looks good, but just wondering do we need to add `div, li, h2` as a part of css identifiers?

Reply:
By doing so, I feel confident they're at the top of the checkout CSS hierarchy. 😂